website/content/docs/secrets/consul.mdx

```diff
@@ -25,41 +25,52 @@ management tool.
     By default, the secrets engine will mount at the name of the engine. To
     enable the secrets engine at a different path, use the `-path` argument.
 
-1.  Bootstrap the Consul ACL system if not already done. To begin configuring the secrets engine, we must give Vault
-    the necessary credentials to manage Consul.
+1. Vault can bootstrap the ACL system of your Consul cluster if it has
```

Comment:
Appreciate the docs update included! I think that due to the way this new feature will create the bootstrap token _and_ store it in Vault without exposing it to the user, we'll need to elaborate a bit more about this process.

I made a task for myself to add more documentation at a later date for this, but I'm happy with this current version for now!